src/AuxDataSchema.h

```diff
@@ -84,13 +84,33 @@ namespace gtirb
             typedef std::map<gtirb::UUID, std::string> Type;
         };
 
+        struct FlaggedSections
+        {
+            static constexpr const char* Name = "FlaggedSections";
+            typedef std::map<gtirb::UUID, std::tuple<gtirb::UUID, uint8_t>> Type;
+        };
+
         /// \brief Auxiliary data covering ELF section properties.
         struct ElfSectionProperties
         {
             static constexpr const char* Name = "elfSectionProperties";
             typedef std::map<gtirb::UUID, std::tuple<uint64_t, uint64_t>> Type;
         };
 
+        struct AllElfSectionProperties
```

Comment:
Why are three `.*ElfSectionProperties` auxiliary tables needed? In particular, if `AllElfSectionProperties` is just a union of the other two, what's the benefit of keeping it?

For the ones that stay, we should add `/// \brief` style comments at some point to clarify the distinction.